tests/test_crawler.py

```diff
@@ -119,28 +119,6 @@ async def test_catalog_deleted_with_new_url(setup_catalog, db, rmock, event_loop
     assert ("PUT", URL(udata_url)) in rmock.requests
 
 
-async def test_udata_connection_error_500(setup_catalog, mocker, analysis_mock, rmock, event_loop):
```

Comment:
Those tests don't make sense anymore IMHO, since we're now bubbling up connection errors from udata connection ("protected" by a worker task).